test/e2e/gateway/tlsroute_test.go

```diff
@@ -72,37 +67,14 @@ func testTLSRoutePassthrough(namespace string) {
 		require.Never(f.T(), func() bool {
 			_, err := f.HTTP.SecureRequest(&e2e.HTTPSRequestOpts{
 				Host: "something.else.not.matching",
-				TLSConfigOpts: []func(*tls.Config){
```

Comment:
this line is redundant since the SNI gets set to the Host if not otherwise specified, so just cleaning this up.